mw-webapp/src/globalStore/ServiceWorkerStore.ts

```diff
@@ -121,7 +122,7 @@ class ServiceWorker {
    * Update current permissions in instance
    */
   private getNotificationPermissions = () => {
-    const isNotificationAvailable = "Notification" in window;
+    const isNotificationAvailable = localStorageWorker.getItemByKey<SettingPageSettings>("settingPage")?.isNotificationAllowed;
```

Comment:
Also I see "?" operator

It strange, as I remember we don't need to use it working with localStorageWorker, otherwise there is no sense in worker, easier to work with localstorage directly

Reply:
Done